functions/__z.fish

```diff
@@ -144,6 +144,7 @@ function __z -d "Jump to a recent directory."
         # Handle list separately as it can print common path information to stderr
         # which cannot be captured from a subcommand.
         command awk -v t=(date +%s) -v list="list" -v typ="$typ" -v q="$q" -F "|" $z_script "$Z_DATA"
+        return
     else
```

Comment:
Looks good, I'd remove the else and just unindent what's inside.

Reply:
Ah yes. Done :+1: